phc-rails/db/schema.rb

```diff
@@ -11,31 +11,44 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150213044554) do
+ActiveRecord::Schema.define(version: 20150309035653) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
   enable_extension "pg_trgm"
 
-  create_table "last_modifieds", force: true do |t|
+  create_table "accounts", force: true do |t|
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.string   "last_modified"
+    t.string   "sf_id"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.